spike/composes.yaml

```diff
@@ -0,0 +1,7 @@
+subgraphs:
+  films:
+    url: 'https://films.example.com'
```

Comment:
Thoughts on this being `routing_url` to match the subgraph commands and to make it distinct from an introspection url?

Reply:
smart!!!!